static/app/components/onboarding/frameworkSuggestionModal.tsx

```diff
@@ -238,53 +252,52 @@ export function FrameworkSuggestionModal({
         <Description>{languageDescriptions[selectedPlatform.key]}</Description>
         <StyledPanel>
           <StyledPanelBody>
-            <Frameworks>
-              {listEntries.map((framework, index) => {
-                const frameworkCategory =
+            <PlatformList>
+              {listEntriesWithVanilla.map((platform, index) => {
+                const platformCategory =
                   categoryList.find(category => {
-                    return category.platforms?.has(framework.id);
+                    return category.platforms?.has(platform.id);
                   })?.id ?? 'all';
 
                 return (
-                  <Framework key={framework.id}>
+                  <PlatformListItem key={platform.id}>
                     <RadioLabel
                       index={index}
                       onClick={() =>
                         setSelectedFramework({
-                          key: framework.id,
-                          type: framework.type,
-                          language: framework.language,
-                          category: frameworkCategory,
+                          key: platform.id,
+                          type: platform.type,
+                          language: platform.language,
+                          category: platformCategory,
+                          link: platform.link,
+                          name: platform.name,
                         })
                       }
                     >
                       <RadioBox
                         radioSize="small"
-                        checked={selectedFramework?.key === framework.id}
+                        checked={selectedFramework?.key === platform.id}
                         readOnly
                       />
-                      <FrameworkIcon size={24} platform={framework.id} />
-                      {framework.name}
+                      <PlatformListItemIcon size={24} platform={platform.id} />
+                      {platform.name}
                     </RadioLabel>
-                  </Framework>
+                  </PlatformListItem>
                 );
               })}
-            </Frameworks>
+            </PlatformList>
           </StyledPanelBody>
         </StyledPanel>
       </Body>
       <Footer>
-        <Actions>
-          <Button onClick={handleSkip}>{t('Skip')}</Button>
-          <Button
-            priority="primary"
-            onClick={handleConfigure}
-            disabled={!selectedFramework}
-            title={!selectedFramework ? t('Select a framework to configure') : undefined}
-          >
-            {t('Configure SDK')}
-          </Button>
-        </Actions>
+        <Button
+          priority="primary"
+          onClick={
+            selectedFramework?.key === selectedPlatform.key ? handleSkip : handleConfigure
```

Comment:
In the skip logic, we trigger the `onboarding.select_framework_modal_skip_button_clicked` analytic event. We could update it to reflect the change, or simply leave it as is, even though the skip option is removed.